frontend/src/redux/roadmaps/types.ts

```diff
@@ -33,6 +33,10 @@ export interface teamMemberRequest {
   type?: RoleType | null;
 }
 
+export interface CheckableUser extends RoadmapUser {
+  checked: boolean;
+}
```

Comment:
This definition could be used also in https://github.com/Vincit/VISDOM-Roadmapper/blob/ec59d1b6e4f645cec8be0e6284e7868c505f1a62/frontend/src/components/modals/EditCustomerModal.tsx#L33